std_msgs/Header header
float64[49] mass_matrix
float64[42] jacobian_matrix
float64[49] m1
float64[49] m2